addon/components/map-commands-dialogs/export.js

```diff
@@ -496,6 +497,85 @@ let FlexberryExportMapCommandDialogComponent = Ember.Component.extend({
   ),
 
   /**
+    Get map layers info.
+
+    @type Array
+    @private
+  */
+  _getLayersInfo: function(layers) {
+    var res = [];
+
+    for (var i = 0; i < layers.length; i++) {
```

Comment:
It is better to use forEach loop, instead of accessing by index in every string.